packages/ag-charts-community/src/chart/axis/axis.ts

```diff
@@ -26,7 +26,7 @@ import { Selection } from '../../scene/selection';
 import { Line } from '../../scene/shape/line';
 import type { TextSizeProperties } from '../../scene/shape/text';
 import { Text, measureText, splitText } from '../../scene/shape/text';
-import { jsonDiff } from '../../sparklines-util';
+import { findMinMax, findRangeExtent, jsonDiff } from '../../sparklines-util';
```

Comment:
Nit: `sparklines-util` is an export bundle, we shouldn't import from it (or the other bundles at that level) internally in `ag-charts-community`.